jdk/src/share/legal/freetype.md

```diff
@@ -0,0 +1,537 @@
+## The FreeType Project: Freetype v2.9
```

Comment:
The "legal" folder is used in jdk9+, for jdk8 licenses should be placed in THIRD_PARTY_README file. We have several of them in jdk8, but after https://github.com/openjdk/jdk8u-dev/pull/557 integration there will be only one.

Reply:
I've copied the entire contents of this file to the THIRD_PARTY_README file. Hopefully that's the correct way to do it.